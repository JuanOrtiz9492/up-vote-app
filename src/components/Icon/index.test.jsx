import React from 'react';
import { render, cleanup } from '@testing-library/react';
import {
  ThumbsUpIcon,
  ThumbsDownIcon,
  SearchIcon,
  WikipediaIcon,
  FacebookIcon,
  TwitterIcon
} from './IconList';

describe('Iconlist Components', () => {

    afterEach(cleanup);

    it('ThumbsUpIcon should Match the Snapshot', () => {
      const { container } = render(<ThumbsUpIcon />);
      expect(container).toMatchSnapshot();
    })

    it('ThumbsDownIcon should Match the Snapshot', () => {
      const { container } = render(<ThumbsDownIcon />);
      expect(container).toMatchSnapshot();
    })

    it('SearchIcon should Match the Snapshot', () => {
      const { container } = render(<SearchIcon />);
      expect(container).toMatchSnapshot();
    })

    it('WikipediaIcon should Match the Snapshot', () => {
      const { container } = render(<WikipediaIcon />);
      expect(container).toMatchSnapshot();
    })

    it('FacebookIcon should Match the Snapshot', () => {
      const { container } = render(<FacebookIcon />);
      expect(container).toMatchSnapshot();
    })

    it('TwitterIcon should Match the Snapshot', () => {
      const { container } = render(<TwitterIcon />);
      expect(container).toMatchSnapshot();
    })
})

