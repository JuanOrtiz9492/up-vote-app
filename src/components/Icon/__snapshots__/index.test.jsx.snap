// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Iconlist Components FacebookIcon should Match the Snapshot 1`] = `
<div>
  <svg
    class="sc-AykKG eHnaMZ"
  >
    facebook-square.svg
  </svg>
</div>
`;

exports[`Iconlist Components SearchIcon should Match the Snapshot 1`] = `
<div>
  <svg
    class="sc-AykKE bLJbQr"
  >
    search.svg
  </svg>
</div>
`;

exports[`Iconlist Components ThumbsDownIcon should Match the Snapshot 1`] = `
<div>
  <svg
    class="sc-AykKD fVEUey"
  >
    thumbs-down.svg
  </svg>
</div>
`;

exports[`Iconlist Components ThumbsUpIcon should Match the Snapshot 1`] = `
<div>
  <svg
    class="sc-AykKC kfAMsF"
  >
    thumbs-up.svg
  </svg>
</div>
`;

exports[`Iconlist Components TwitterIcon should Match the Snapshot 1`] = `
<div>
  <svg
    class="sc-AykKH xriyS"
  >
    twitter.svg
  </svg>
</div>
`;

exports[`Iconlist Components WikipediaIcon should Match the Snapshot 1`] = `
<div>
  <svg
    class="sc-AykKF iRiTbg"
  >
    wikipedia-w.svg
  </svg>
</div>
`;
